src/query/sql/src/planner/binder/table.rs

```diff
@@ -565,9 +573,191 @@ impl Binder {
                     .await
             }
             TableReference::Join { .. } => unreachable!(),
+            TableReference::Values {
+                span,
+                values,
+                alias,
+            } => self.bind_values(bind_context, *span, values, alias).await,
         }
     }
 
+    #[async_backtrace::framed]
+    pub(crate) async fn bind_values(
+        &mut self,
+        bind_context: &BindContext,
```

Comment:
You can change it to `bind_context: &mut BindContext`, because `self.bind_values(bind_context, *span, values, alias).await,` the `bind_context` is `&mut`